docs/plugin-development-guide/naming.rst

```diff
@@ -26,6 +26,10 @@ to reflect your plugin functionality. Basing on the vendor and plugin names esta
 
     * ``\Acme\SyliusExamplePlugin\AcmeSyliusExamplePlugin()`` -> ``\IronMan\SyliusProductOnDemandPlugin\SyliusProductOnDemandPlugin()``
 
+* In ``tests/Application/config/bundles.php``:
+
+* ``Acme\SyliusExamplePlugin\AcmeSyliusExamplePlugin::class`` -> ``IronMan\SyliusProductOnDemandPlugin\SyliusProductOnDemandPlugin::class``
```

Comment:
I believe it should replace the outdated previous point (there is no `tests/Application/Kernel.php` anymore). Also, this line should be indented as others 🖖